src/app/lib/analyticsUtils/index.test.js

```diff
@@ -342,3 +343,37 @@ describe('sanitise', () => {
     expect(sanitise('hi hello there')).toEqual('hi+hello+there');
   });
 });
+
+describe('getPublishedDatetime', () => {
+  const data = {
+    metadata: {
+      firstPublished: 946688461000,
+      seconds: 946688461,
+      invalidDate: 'foobar',
+    },
+  };
+
+  it('should find value in good data', () => {
+    const publishedTime = getPublishedDatetime('firstPublished', data);
+
+    expect(publishedTime).toEqual('2000-01-01T01:01:01.000Z');
+  });
+
+  it('should autodetect timestamp in seconds and autocorrect', () => {
+    const secondsTimestamp = getPublishedDatetime('seconds', data);
+
+    expect(secondsTimestamp).toEqual('2000-01-01T01:01:01.000Z');
```

Comment:
Can you change this timestamp under `seconds` slightly please, if the function was hardcoded to `firstPublished` it would also pass